### PR TITLE
Result suggest

### PR DESCRIPTION
**Pull request by julianecasier**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Routes, Route } from "react-router-dom";
 import ResultsCategory from "@components/ResultsCategory";
+import SuggestionResultPage from "@pages/SuggestionResultPage";
 import Home from "./pages/Home";
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Attention, blank line

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/suggestion" element={<Suggestion />} />
         <Route path="/results/:category" element={<ResultsCategory />} />
+        <Route path="/suggestion/results" element={<SuggestionResultPage />} />
```
Comment (LoicBrassart) [inline frontend/src/App.jsx]: Indentation is off?

File: frontend/src/pages/SuggestionResultPage.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import SugResult from "../components/SugResult";
+
+export default function SuggestionResultPage() {
+  return (
+    <>
```
Comment (LoicBrassart) [inline frontend/src/pages/SuggestionResultPage.jsx]: Useless Fragment (it's wrap ping à solo jsxnode)

File: frontend/src/pages/Suggestion.jsx [hunk 4]
```diff
@@ -1,5 +1,9 @@
 import QuestionSug from "../components/QuestionSug";
 
 export default function Suggestion() {
-  return <QuestionSug />;
+  return (
+    <>
```
Comment (LoicBrassart) [inline frontend/src/pages/Suggestion.jsx:5]: Useless fragment, it's wrap ping à solo El e ment

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Clean up your fragments ;-)

**Review by ghost — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Better!

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**